migrations/add_rating_table_5.sql

```diff
@@ -0,0 +1,20 @@
+/* Add ratings table and reviewId foreign key contstraint */

```

Comment:
I did it this way only because that's the way the migrations were being done before, do you want me to redo the format of all the migrations? I don't mind doing that of course since it should be very quick, just let me know and I'll cleanup the migrations folder.

Reply:
That would be fantastic if you would do that, thanks!